CRM/Contribute/Page/ContributionPage.php

```diff
@@ -675,7 +675,7 @@ public function pagerAtoZ($whereClause, $whereParams) {
 SELECT DISTINCT UPPER(LEFT(title, 1)) as sort_name
 FROM civicrm_contribution_page
 WHERE $whereClause
-ORDER BY LEFT(title, 1)
+ORDER BY UPPER(LEFT(title, 1))
```

Comment:
It's probably OK here - after all it was an unindexed query anyway due to the LEFT but we need to be careful with UPPER & LOWER because they do break indexing